src/styles/LESS/rental-pane.less

```diff
@@ -0,0 +1,20 @@
+.rental-price-tab {
```

Comment:
Good idea to separate it into its own less file--we should do this with the other panes in the detail-page. I see how confusing it can be and how big it can get.